src/commands/cluster-keyslot.json

```diff
@@ -4,16 +4,36 @@
         "complexity": "O(N) where N is the number of bytes in the key",
         "group": "cluster",
         "since": "3.0.0",
-        "arity": 3,
+        "arity": -3,
         "container": "CLUSTER",
         "function": "clusterCommand",
         "command_flags": [
-            "STALE"
```

Comment:
why was the STALE flag removed?
also READONLY means that the command reads from the keyspace, but this command doesn't (it doesn't look up the key, and doesn't care if it exists or not)